packages/fx-core/resource/package.nls.json

```diff
@@ -210,6 +210,9 @@
   "_error.appstudio.BotProvisionReturnsForbiddenResult.comment": "This is to describe API call, no need to translate 'Botframework'.",
   "error.appstudio.BotProvisionReturnsConflictResult": "Botframework provisioning returns conflict result when attempting to create bot registration.",
   "_error.appstudio.BotProvisionReturnsConflictResult.comment": "This is to describe API call, no need to translate 'Botframework'.",
+  "error.appstudio.localizationFile.pathNotDefined": "Localization file not found. Path: %s.",
+  "error.appstudio.localizationFile.validationFailed": "Localization file validation failed. File: %s. Error:\n%s",
+  "error.appstudio.localizationFile.validationException": "Localization file validation failed with exceptions. File: %s. Error: %s",
```

Comment:
Proposed content: Unable to validate localization file due to exceptions. File: %s. Error: %s"